rachit-1797/README.md

```diff
@@ -0,0 +1,24 @@
+# 100DaysOfCode
+
+**Rachit Gupta**
+
+*Second Year CSE, MNIT Jaipur*
+
+---
+
+## Goals
+
+**Competitive Programming**
+- [] Increase [codechef] (https://www.codechef.com/users/rachit1797) rating to 1800
+- [] Increase [codeforces] (https://codeforces.com/profile/rachit1797) rating to 1600
```

Comment:
Add a space in between all the square boxes `[]` for checkbox: the syntax is `-<space>[<space>]`
For links, remove the space between the different types of brackets. The syntax is `[text](url)`

Be sure to preview your changes.